scrnaseq.Rmd

```diff
@@ -31,6 +31,8 @@ options(stringsAsFactors=FALSE,
 reticulate_python3_path = unname(Sys.which("python3"))
 Sys.setenv(RETICULATE_PYTHON = reticulate_python3_path)
 
+Sys.setenv(RETICULATE_PYTHON = "C:/Program Files/Python38/python.exe")
```

Comment:
This looks like we have 2 rows setting RETICULATE_PYTHON?

Reply:
Its custom and I will delete it